test/features/schema_tests.js

```diff
@@ -11,6 +11,11 @@ var Ajv = require('ajv');
 describe('Responses should conform to the provided JSON schema of the responce', function() {
     var ajv = new Ajv({});
 
+    function getToday() {
+        const now = new Date();
+        return `${now.getUTCFullYear()}/${now.getUTCMonth() + 1}/${now.getUTCDate()}`;
```

Comment:
I see, good point. Should we actually accept /2016/9/1/ rather than /2016/09/01/ ?